Makefile

```diff
@@ -18,15 +18,15 @@ test-pdb: test
 .PHONY: test lint coverage
 
 lint:
-	flake8 --extend-exclude "*/migrations/,local_settings.py" devel main mirrors news packages releng templates todolists visualize *.py
+	uv run flake8 --extend-exclude "*/migrations/,local_settings.py" devel main mirrors news packages releng templates todolists visualize *.py
```

Comment:
should re-evaluate if ruff already has the formatting rules, or ugh `ruff format` I guess.